backend/cellguide/pipeline/__init__.py

```diff
@@ -1,14 +1,18 @@
 import sys
 import time
 
+from backend.cellguide.pipeline.metadata import run as run_metadata_pipeline
 from backend.cellguide.pipeline.ontology_tree import run as run_ontology_tree_pipeline
 
 
 def run_cellguide_pipeline():
     output_directory = f"cellguide_pipeline_output__{int(time.time())}"
 
     # Run ontology tree pipeline
-    run_ontology_tree_pipeline(output_directory)
+    ontology_tree = run_ontology_tree_pipeline(output_directory)
+
+    # Generate cell guide cards, synonyms, and descriptions
+    run_metadata_pipeline(ontology_tree)
```

Comment:
Ah, I missed this, but are you outputting the resulting JSON from `run_metadata_pipeline`?